### DISCUSSION TITLE
Force and Stress Values in Structure Relaxation

### DISCUSSION THREAD

**Discussion by Bramantya98** [CederGroupHub/chgnet#214] [Q&A]
Dear all,

I'm new on using CHGNet and still exploring its features. I was wondering if it's possible to include force and stress values in the output during structure optimization. Following the examples in the notebooks, the output give information about energy and magnetic moment information for each atom. It would be helpful to have a more comprehensive output, including force and stress values, similar to what’s provided in a static calculation (prediction). Thank you!

**Comment by bowen-bd** [comment]
They're in the trajectory in the output
